### PR TITLE
inadyn: create addon

### PR DESCRIPTION
**Pull request by awiouy**
Hello!

Here is my first addon for LibreELEC. I hope I got it right.

Thank you for your attention, and regards
@

### CODE REVIEW

**Comment by lrusak** [comment]
Looks fine to me 👍

**Comment by lrusak** [comment]
please updated the headers to something current such as https://github.com/LibreELEC/LibreELEC.tv/blob/master/tools/distro-tool#L2-L18

File: packages/addons/service/inadyn/package.mk [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+PKG_ARCH="any"
+PKG_LICENSE="GPLv2"
+PKG_SITE="http://troglobit.com/inadyn.html"
+PKG_URL="ftp://troglobit.com/$PKG_NAME/$PKG_NAME-$PKG_VERSION.tar.xz"
+PKG_DEPENDS_TARGET="toolchain libressl"
+PKG_PRIORITY="optional"
+PKG_SECTION="service/system"
+PKG_SHORTDESC="Inadyn, a small and simple DDNS client"
+PKG_LONGDESC="Inadyn is a small and simple DDNS client with HTTPS support. It is commonly available in many GNU/Linux distributions, used in off-the-shelf routers and Internet gateways to automate the task of keeping your DNS record up to date with any IP address changes from your ISP. It can also be used in installations with redundant (backup) connections to the Internet."
+PKG_DISCLAIMER="This is a community addon. Do therefore not expect support from LibreLEC forum and irc channel."
```
Comment (lrusak) [inline packages/addons/service/inadyn/package.mk]: This line can be removed

File: packages/addons/service/inadyn/package.mk [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+PKG_MAINTAINER="Anton Voyl (awiouy at gmail.com)"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_TYPE="xbmc.service"
+PKG_ADDON_PROVIDES=""
+PKG_ADDON_REPOVERSION="7.0"
```
Comment (lrusak) [inline packages/addons/service/inadyn/package.mk]: Please update to 8.0 unless you meant to PR to libreelec-7.0 branch

File: packages/addons/service/inadyn/changelog.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+7.0.100
```
Comment (lrusak) [inline packages/addons/service/inadyn/changelog.txt]: please set to 8.0.100

**Comment by lrusak** [comment]
@awiouy please update with the last comments. I can take care of the backport to libreelec-7.0

**Comment by chewitt** [comment]
icon to use: http://chewitt.libreelec.tv/test/icon_inadyn.png

**Comment by awiouy** [comment]
Thank you for the feedback and the icon. I updated the addon accordingly. Please commit.
Regards, @

**Comment by CvH** [comment]
@awiouy minor nitpicking from me, could you cut the changelog to start with 7.0 ? (at 7.0 it isn't that important what happens at 4.0 and 6.0 ;) )

**Comment by awiouy** [comment]
Hi CvH,
I started the changelog at 8.0.100, which is the first initial addon for LibreELEC.
Please backport the addon to LE 7.0, since it fixes a bug introduced on OE 7.0.
Thank you, and regards,
@

**Pull request merged**